### PR TITLE
Post GA fixes - OCP 3 to 4 migration

### PR DESCRIPTION
**Pull request by apinnick**
Added slash to `/master1.example.com/` directory created by cpma

### CODE REVIEW

File: modules/migration-using-cpma.adoc [hunk 1]
```diff
@@ -87,7 +87,7 @@ INFO[29 Aug 19 00:08 UTC] Successfully finished transformations
 The CPMA creates the following files and directory in the current directory if you did not specify an output directory:
 
 * `cpma.yaml` file: Configuration options that you provided when you ran the CPMA
-* `/master1.example.com`: Configuration files from the master node
+* `/master1.example.com/`: Configuration files from the master node
```
Comment (gildub) [inline modules/migration-using-cpma.adoc]: There is no slash at the beginning.

**Comment by apinnick** [comment]
@bergerhoffer This is a minor fix to the version we released yesterday. Can you please review and merge? Thanks.

File: modules/migration-installing-migration-operator-manually.adoc [hunk 2]
```diff
@@ -14,57 +14,49 @@ You can install the CAM Operator manually on an {product-title} 3 source cluster
...
 
 * You must have `podman` installed.
+* You must have access to the link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry].
 
 .Procedure
 
+. Log in to the Red Hat Container Registry with your Customer Portal credentials:
++
+----
+# podman login registry.redhat.io
```
Comment (bergerhoffer) [inline modules/migration-installing-migration-operator-manually.adoc]: Does this really need to be run as root (with the # prompt)? The next one uses $, so I'm assuming this prompt should be $ as well.

File: modules/migration-installing-migration-operator-manually.adoc [hunk 3]
```diff
@@ -14,57 +14,49 @@ You can install the CAM Operator manually on an {product-title} 3 source cluster
 .Prerequisites
 
 * You must have `podman` installed.
+* You must have access to the link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry].
 
 .Procedure
 
+. Log in to the Red Hat Container Registry with your Customer Portal credentials:
```
Comment (bergerhoffer) [inline modules/migration-installing-migration-operator-manually.adoc]: Minor, but suggest "Red Hat Customer Portal"

File: modules/migration-installing-migration-operator-manually.adoc [hunk 4]
```diff
@@ -14,57 +14,49 @@ You can install the CAM Operator manually on an {product-title} 3 source cluster
...
 ----
 $ podman cp $(podman create registry.redhat.io/rhcam-1-0/openshift-migration-rhel7-operator:v1.0 ):/operator.yml ./
 ----
 
 . Download the `controller-3.yml` file:
-//
-// * {product-title} 3:
 +
 ----
 podman cp $(podman create registry.redhat.io/rhcam-1-0/openshift-migration-rhel7-operator:v1.0 ):/controller-3.yml ./
```
Comment (bergerhoffer) [inline modules/migration-installing-migration-operator-manually.adoc]: I know this was already here, but do you mind adding the $ prompt here too please?

**Review by bergerhoffer — COMMENTED** [review]
Just minor things, but would you also mind squashing please? Thanks!

File: modules/migration-installing-migration-operator-manually.adoc (hunk 2 above)
Comment (apinnick) [reply modules/migration-installing-migration-operator-manually.adoc]: I was wondering that myself. It's root in the documentation that I linked to. Probably safe to change to $.

**Comment by jwmatthews** [comment]
Technical changes look good from my side.

**Pull request merged**